nucypher/cli/commands/alice.py

```diff
@@ -483,6 +484,13 @@ def grant(general_config,
         bob_encrypting_key = card.encrypting_key.hex()
         bob_verifying_key = card.verifying_key.hex()
 
+    if not bob_encrypting_key or bob_verifying_key:
+        emitter.message("*Caution: Only enter public keys*")
+    if not bob_encrypting_key:
+        bob_encrypting_key = click.prompt("Enter Bob's encrypting key")
+    if not bob_verifying_key:
+        bob_verifying_key = click.prompt("Enter Bob verifying key")
+
```

Comment:
Do we need to be careful here of `force` is true, or is that case handled earlier?

Reply:
:heavy_check_mark: